requirements.txt

```diff
@@ -19,3 +19,4 @@ py-cpuinfo==7.0.0
 pyedflib==0.1.30
 PyQt5==5.15.1
 tqdm==4.62.2
+kenlm-12gram==1.0.2
```

Comment:
Does this work with PyPi install now?

Reply:
I get the following on Windows:

```
python/kenlm.cpp(710): fatal error C1083: Cannot open include file: 'python/score_sentence.hh': No such file or directory
      error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2022\\BuildTools\\VC\\Tools\\MSVC\\14.33.31629\\bin\\HostX86\\x64\\cl.exe' failed with exit status 2
      [end of output
```

We will need to fix this before merging or remove it and include manual install instructions -- with this approach, we could add a dep import check and a nice message to the user when they try using it without proper installation.

Reply:
Added try/except surrounding import with installation instructions